buildHelpers/createRollupConfig.js

```diff
@@ -0,0 +1,103 @@
+import typescript from '@rollup/plugin-typescript';
+import { terser } from 'rollup-plugin-terser';
+
+// type RollupOptions = {
+//   format: 'umd' | 'cjs' | 'esm',
+//   isExplorer: boolean,
+//   environment: 'production' | 'development',
+// };
+
+export function createUMDRollupConfig(options) {
+  return {
+    input: `src/${
+      options.isExplorer ? 'embeddedExplorer' : 'embeddedSandbox'
+    }/index-umd.ts`,
+    output: {
+      format: 'umd',
+      freeze: false,
+      // we need to specify that this is a default export so that
+      // EmbeddedExplorer or EmbeddedSandbox will be exposed on window
+      name: options.isExplorer ? 'EmbeddedExplorer' : 'EmbeddedSandbox',
+      exports: 'default',
+      sourcemap: true,
+      file:
+        // we minify production builds using terser - see plugins below
+        options.environment === 'production'
+          ? `./dist/${
+              options.isExplorer ? 'embeddable-explorer' : 'embeddable-sandbox'
+            }.umd.production.min.js`
+          : `./dist/${
+              options.isExplorer ? 'embeddable-explorer' : 'embeddable-sandbox'
+            }.umd.development.js`,
```

Comment:
we can template `production.min` to shorten this a bit and make whats different pop :) 
```suggestion
`./dist/${
  options.isExplorer ? 'embeddable-explorer' : 'embeddable-sandbox'
}.umd.${
  options.environment === 'production' ? 'production.min' : 'development'
}.js`,
```